src/commands/scaling/addScaleRule/queue/QueueNameStep.ts

```diff
@@ -3,24 +3,20 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import type { ContainerApp, ScaleRule } from '@azure/arm-appcontainers';
 import { AzureWizardPromptStep } from '@microsoft/vscode-azext-utils';
 import { localize } from '../../../../utils/localize';
 import type { IAddScaleRuleContext } from '../IAddScaleRuleContext';
 
 export class QueueNameStep extends AzureWizardPromptStep<IAddScaleRuleContext> {
-    containerApp: ContainerApp | undefined;
-    scaleRules: ScaleRule[] | undefined;
-
     public async prompt(context: IAddScaleRuleContext): Promise<void> {
         context.queueName = (await context.ui.showInputBox({
             prompt: localize('queueNamePrompt', 'Enter a name for the queue.'),
-            validateInput: (value: string | undefined): string | undefined => this.validateInput(value)
+            validateInput: this.validateInput
         })).trim();
     }
 
     public shouldPrompt(context: IAddScaleRuleContext): boolean {
-        return context.queueName === undefined;
+        return !context.queueName;
```

Comment:
Changed a lot of these to just a `!` because I feel like there are other falsey values besides `undefined` that I would like to prevent as valid input.